apps/omg/lib/omg/internal_event_bus.ex

```diff
@@ -26,11 +26,18 @@ defmodule OMG.InternalEventBus do
 
   Message: {:internal_event_bus, :enqueue_block, OMG.Block.t()}
 
+  #### `preprocess_emit_events`
+
+  Decoupling API from Watcher core - Enrich data before broadcast
+
+  Message: {;internal_event, :preprocess_emit_events, list()}
```

Comment:
a `;` typo, and whoops, there's a similar one just below, which I guess this originates from